airgun/widgets.py

```diff
@@ -334,10 +334,12 @@ def read(self):
 class Search(Widget):
     search_field = TextInput(locator=(
         ".//input[@id='search' or @ng-model='table.searchTerm' or "
-        "contains(@ng-model, 'Filter')]"))
+        "contains(@ng-model, 'Filter') or "
+        "starts-with(@id, 'downshift-')]"))
```

Comment:
that is not good. If you have specific locator, so override that class for your view. Never expand most fundamental classes in framework like this